### PR TITLE
KBadge redesign

### PR DESCRIPTION
**Pull request by Pan-Bubr**
### Summary
KBadge was redesigned to allow for rectangular border shape, new standardized color and a different paddings

#### Changes made:
* Add new "info" appearance for badge
* Add round and rectangular shape for badge
* Change paddings
* Add variables for line height
* Update test snapshots

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: docs/components/badge.md [hunk 1]
```diff
@@ -13,24 +13,39 @@ something.
...
+<KBadge appearance="warning">Round</KBadge>
+<KBadge appearance="warning" shape="rectangular">Rectangular</KBadge>
 
 ```vue
-<KBadge appearance="success">HIGH</KBadge>
-<KBadge appearance="warning">MEDIUM</KBadge>
-<KBadge appearance="danger">LOW</KBadge>
-<KBadge>DEFAULT BADGE</KBadge>
+<KBadge appearance="warning">Round</KBadge>
+<KBadge appearance="warning" shape="rectangle">Rectangular</KBadge>
```
Comment (kaiarrowood) [inline docs/components/badge.md:48]:
```suggestion
<KBadge appearance="warning" shape="rectangular">Rectangular</KBadge>
```

**Comment by kaiarrowood** [comment]
@Pan-Bubr I see this error when I go to the Badge page, can you debug?

image

File: packages/KBadge/KBadge.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    :class="[appearance !== 'custom' && `kbadge-${appearance}`]"
+    :class="[appearance !== 'custom' && `kbadge-${appearance}`, `kbadge-${shape}`]"
```
[the remark this answers is not in the document]
Comment (Pan-Bubr) [reply packages/KBadge/KBadge.vue:3]: I decided to simplify it even more. It turns out that we do not need that conditional here.

File: docs/components/badge.md (hunk 1 above)
Comment (Pan-Bubr) [reply docs/components/badge.md:48]: Fixed and also resolved the comment with a Vue warning as well.

**Review by kaiarrowood — APPROVED** [review]
Looks great! Nice work 🐉 🔥

**Pull request merged**